### PR TITLE
Migration touch-ups

### PR DESCRIPTION
**Pull request by e-papanicolas**
This fixes broken section nav links on the main page. Added the # to make sure they don't redirect to a 404, and instead just scroll to their related sections on the page. It fixes some punctuation on the Donate FAQ's, and changes the donorbox link to open in a new tab. It adds "check another conviction" button text and link to sanity config.

### CODE REVIEW

**Review by ShepleySound — COMMENTED** [review]
Good catch on the `#` tags for the links. 

For the calculator nav, let's remove it from the navItems file and keep it as is, since design wanted the button styling. Although, adding the Button that currently exists in the desktop nav to the bottom of the mobile nav would work!

File: pages/calculator/[slug].js [hunk 1]
```diff
@@ -21,6 +23,7 @@ import ExternalButton from "../../components/ExternalButton";
 import portableTextComponents from "../../utils/portableTextComponents";
 
 export default function CalculatorSlugRoute({ page, calculatorConfig }) {
+  console.log(page);
```
Comment (e-papanicolas) [inline pages/calculator/[slug].js:26]: We should delete this console.log!

**Comment by e-papanicolas** [comment]
Amazing! This looks so good, I can see all of the button logic, and things make more sense. Thanks for working on this :)

**Review by ShepleySound — APPROVED** [review]
This all looks good to me! I've made some changes in Sanity Studio and confirmed that the feedback forms are going to the right place. Great work!

**Pull request merged**